content/guides/component/src/routes.clj

```diff
@@ -0,0 +1,13 @@
+ ;; tag::ns[]
+(ns routes)
+ ;; end::ns[]
+
+ ;; tag::response[]
+(defn respond-hello [request]
+  {:status 200 :body "Hello, world!"})
+ ;; end::response[]
+
+ ;; tag::routes[]
+(def routes
```

Comment:
I have a concern on this definition. I almost always want my routes to include interceptors that have to be built with config information. (E.g., attaching a Datomic connection... need the URI.)

So most of the time, I make `routes` a function not a var.

I think in this case that would also mean making _another_ Component for the routes, or having the Pedestal component build the routes and inject them into the service map. Either one may be too much for this guide, but maybe it's worth a mention in the text.